src/stream.rs

```diff
@@ -78,6 +78,13 @@ manual_braid! {
 }
 impl_extra!(IgdbId, IgdbIdRef);
 
+manual_braid! {
+    /// A content classification label ID
+    pub struct CclId;
+    pub struct CclIdRef;
+}
+impl_extra!(CclId, CclIdRef);
+
```

Comment:
should this be an enum?

Reply:
Good point. And we already have that enum 🙃 (`ContentClassificationId`')